packages/react-native-renderer/src/ReactNativeTypes.js

```diff
@@ -4,7 +4,8 @@
  * This source code is licensed under the MIT license found in the
  * LICENSE file in the root directory of this source tree.
  *
- * @format
+ * @noformat
```

Comment:
There was some login in the build switch to flip these files to `@noformat` but we can just... make them that way in source.